pkg/daemon/daemon.go

```diff
@@ -387,13 +413,25 @@ func (dn *Daemon) applyNodePtpProfile(runID int, nodeProfile *ptpv1.PtpProfile)
 			configFile = fmt.Sprintf("ptp4l.%d.config", runID)
```

Comment:
Can we just keep a mapping of ptpconfigName: socketPath, and then there's no iterating through again around line 613?

Reply:
I do not want to refactor existing code. Makes hard for any backport fix, We should do iterative update